src/activitypub.js

```diff
@@ -239,3 +239,14 @@ export async function broadcastMessage(bookmark, action, db, account, domain) {
     }
   }
 }
+
+export function synthesizeActivity(note) {
+  return {
+    // Fake activity URI adds a "a-" prefix to the Note/message guid
```

Comment:
ooh, I really like this-- I was thinking I was going to have to double the number of objects in the activitypub.db database.